pkcs8/src/private_key_info.rs

```diff
@@ -123,7 +131,7 @@ impl<'a> Message<'a> for PrivateKeyInfo<'a> {
         F: FnOnce(&[&dyn Encodable]) -> der::Result<T>,
     {
         f(&[
-            &VERSION,
+            &(Version::V1 as u8),
```

Comment:
This should probabyl be consistent with the `&(<version to u8>)` above. (i.e. use `From` / `Into` both times, or `as u8` both times)